cli.go

```diff
@@ -614,6 +622,16 @@ func (cli *CLI) ParseFlags(args []string) (*Config, []string, bool, bool, error)
 		return nil, nil, false, false, err
 	}
 
+	// Post-processing of no-prefix option
+	if no_prefix != nil {
+		for _, p := range *c.Prefixes {
+			p.NoPrefix = no_prefix
```

Comment:
We should probably only set this value if NoPrefix is nil for the prefix, that way it can still act as an override.  Same goes for the secrets below.

Reply:
I think the only time that (NoPrefix not being nil) would come up is if they had the NoPrefix set via the config file. Which would mean that the config file settings would override the command line argument. Isn't it usually the opposite, with the command line arguments overriding the config file?

Reply:
I dug around just to be sure and the command line argument taking precedence is definitely the de-facto cross platform standard. So I think we should keep it as is.

Reply:
Giving the CLI precedence makes sense to me 👍